dataland-qa-service/src/main/kotlin/org/dataland/datalandqaservice/api/QaApi.kt

```diff
@@ -4,8 +4,10 @@ import io.swagger.v3.oas.annotations.Operation
 import io.swagger.v3.oas.annotations.responses.ApiResponse
 import io.swagger.v3.oas.annotations.responses.ApiResponses
 import io.swagger.v3.oas.annotations.security.SecurityRequirement
```

Comment:
In dieser Datei gibt es bei summary/description ein bisschen Inkonsistenz bezüglich der Frage, ob man diese mit einem Punkt beendet oder nicht. Würde ich angleichen.

Reply:
Sollten jetzt überall "." am Ende stehen.